spring-sample-app/src/main/java/io/github/cosminseceleanu/tutorials/sampleapp/user/controller/UserController.java

```diff
@@ -0,0 +1,33 @@
+package io.github.cosminseceleanu.tutorials.sampleapp.user.controller;
+
+import io.github.cosminseceleanu.tutorials.sampleapp.user.representation.UserRepresentation;
+import io.github.cosminseceleanu.tutorials.sampleapp.user.service.UserService;
+import javax.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+
+@RestController
+@AllArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+    @GetMapping("/users/{id}")
+    public Mono<UserRepresentation> get(@PathVariable("id") String id) {
+        String documentaion = "misspelled";
```

Comment:
**[misspell]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>"documentaion" is a misspelling of "documentation"